### DISCUSSION TITLE
Integration for Tuya 16 Zone Sprinkler

### DISCUSSION THREAD

**Discussion by jhouse1989** [make-all/tuya-local#1789] [General]
Hello, I was able to get use your code to get the sprinkler communicating with Home Assistant but only the 1st valve would map. I was not able to get all 16 zones to map as well as use the timer features. Any suggestions to get this working?

This is being sold as Pihode 16 zone sprinkler on Amazon but it also available under many others names on AliExpress.

https://www.amazon.com/Pihode-Sprinkler-Controller-Irrigation-Customized/dp/B0CQ8BYZ5P

**Comment by malosaa** [comment]
What one was selected default ?

What i mean is when u added the device, what device type did you choose ? 
Maybe i can look at it.
